client/console/src/main/java/org/apache/syncope/client/console/panels/ConnObjectListViewPanel.java

```diff
@@ -240,4 +301,27 @@ private AbstractSearchPanel getSearchPanel(final String id, final String anyType
         return new ConnObjectSearchPanel.Builder(resource, anyTypeKind, anyType,
                 new ListModel<>(clauses)).required(true).enableSearch().build(id);
     }
+
+    private AbstractAnyRestClient<? extends AnyTO> getRestClient(final String type) {
```

Comment:
You can safely remove these last two methods, given the changes suggested above.